<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GWEpp: Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBox_minimize_to_tray_on_close">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Minimize to tray on close</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_update_freq">
       <property name="text">
        <string>Update frequency (ms)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBox_minimize_to_tray_on_startup">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Minimize to tray on startup</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_update_freq">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>500</number>
       </property>
       <property name="maximum">
        <number>3000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBox_save_last_fan_and_clock_offset_profiles">
       <property name="text">
        <string>Save last fan and clock offset profiles</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_minimize_to_tray_on_startup</tabstop>
  <tabstop>checkBox_minimize_to_tray_on_close</tabstop>
  <tabstop>spinBox_update_freq</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
